src/behavior.scm

```diff
@@ -23,10 +23,10 @@
 ; use of.
 ;
 ; A general OpenPsi rule has the form of if(context) then take(action);
-; these can contain variables, and can also be classed into different
+; these can contain variables, adn can also be classed into different
 ; groups based on the demands that they are fulfilling.
 ;
-; The content below consits entirely of actions to be taken; the
+; The content below consits entirely of actions to nbe taken; the
```

Comment:
Its even a waste of the time of the other people who get email notifications of pull requests -- its a waste of thier time, too, to read all of these comments ...